manifests/00-crd.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: consoles.console.openshift.io
+  name: consoleoperatorconfigs.console.operator.openshift.io
 spec:
-  group: console.openshift.io
+  scope: Cluster
+  group: console.operator.openshift.io
+  version: v1
   names:
-    kind: Console
-    listKind: ConsoleList
-    plural: consoles
-    singular: console
-  scope: Namespaced
-  version: v1alpha1
+    kind: ConsoleOperatorConfig
+    listKind: ConsoleOperatorConfigList
+    plural: consoleoperatorconfigs
+    singular: consoleoperatorconfig
+    categories:
+    - coreoperators
```

Comment:
`oc get coreoperators -o yaml` should include this, the console operator config.